buildSrc/build.gradle.kts

```diff
@@ -17,8 +17,8 @@ plugins {
 dependencies {
     // This allows usage of kotlin and android gradle plugin apis as well as importing relevant types from the plugins in buildSrc files.
     // Found in https://quickbirdstudios.com/blog/gradle-kotlin-buildsrc-plugin-android/
-    // Unable to use a single source of truth due to issue mentioned in https://github.com/gradle/kotlin-dsl-samples/issues/1320#issuecomment-486309410
+    // Unable to use a   single source of truth due to issue mentioned in https://github.com/gradle/kotlin-dsl-samples/issues/1320#issuecomment-486309410
     // TODO: These should be updated in tandem with source of truth values in Dependencies.kt.
-    implementation("com.android.tools.build:gradle:7.2.0")
-    implementation("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.0")
+    implementation("com.android.tools.build:gradle:7.2.1")
```

Comment:
Hey @LuisRocketRangel, I believe this com.android.tools.build:gradle dependency should be set to the same version as the one in Dependencies.kt

Reply:
@LuisRocketRangel Agree with what @IvanTelisman mentioned here. Both versions should match